src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.SourceBuild.Tasks.PublishVerticalManifest/PublishVerticalManifest.cs

```diff
@@ -0,0 +1,71 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Build.Utilities;
+using Microsoft.Build.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+
+namespace Microsoft.DotNet.SourceBuild.Tasks.PublishVerticalManifest
+{
+    public class PublishVerticalManifest : Task
+    {
+        [Required]
+        public ITaskItem[] AssetManifestPath { get; set; }
+        [Required]
+        public string VerticalAssetManifestOutputPath { get; set; }
```

Comment:
MergedManifestOutputPath is probably a better name here in case we want to reuse this task for other operations as well.

Reply:
I don't think we'll want to reuse it, the logic will be a bit different. Also, I think we'll want to add an extra attribute to `VerticalManifests`, something like `VerticalName` so we can figure out which artifacts to download